htdocs/product/admin/product.php

```diff
@@ -663,15 +663,14 @@
 print '</tr>';
 
 // Activate propal merge produt card
-/* Kept as hidden feature only. PRODUIT_PDF_MERGE_PROPAL can be added manually. Still did not understand how this feature works.
-
+// When a product have PDF files as attachements, when propal is generated, the PDF files will be concatened at the end of the propal
```

Comment:
Does this works for azur and cyan or only azur ?
Does this is done for all generated PDF (even generation done silently when updating the proposal) or just when you click somewhere on a checkbox ?

Reply:
Because, if the feature PRODUIT_PDF_MERGE_PROPAL become public, it must works, for all templates provided by default (so both azur and cyan, i don't know when we will removed azur).